Mathlib/Data/Sigma/Basic.lean

```diff
@@ -247,12 +247,7 @@ protected theorem eq {α : Sort*} {β : α → Sort*} : ∀ {p₁ p₂ : Σ' a,
 theorem «forall» {p : (Σ'a, β a) → Prop} : (∀ x, p x) ↔ ∀ a b, p ⟨a, b⟩ :=
   ⟨fun h a b ↦ h ⟨a, b⟩, fun h ⟨a, b⟩ ↦ h a b⟩
 
-#adaptation_note
-/--
-This should be renamed back to `exists` after `nightly-2024-07-31`.
--/
-@[simp]
-theorem exists' {p : (Σ'a, β a) → Prop} : (∃ x, p x) ↔ ∃ a b, p ⟨a, b⟩ :=
+@[simp] lemma «exists» {p : (Σ' a, β a) → Prop} : (∃ x, p x) ↔ ∃ a b, p ⟨a, b⟩ :=
```

Comment:
AFAIR, several similar lemmas lost `@[simp]` recently, because Lean tries to apply them everywhere. @mattrobball, am I right?